test_vm/src/lib.rs

```diff
@@ -51,9 +57,13 @@ pub struct VM<'bs> {
     actors_dirty: RefCell<bool>,
     actors_cache: RefCell<HashMap<Address, Actor>>,
     empty_obj_cid: Cid,
-    // invocationStack
+    network_version: NetworkVersion,
+    curr_epoch: ChainEpoch,
+    pub invocations: RefCell<Vec<InvocationTrace>>,
```

Comment:
It's strange for this to be public, but nothing else?

Reply:
It's also generally a bad idea to make a `RefCell` public (hard to enforce the "only one mutable borrow at one time" constraint).

Reply:
switched to a function consuming the private field and returning, I think that should be better